docs/docs/reference/Vectors.md

```diff
@@ -188,9 +206,9 @@ Currently there are no runtime parameters available for FLAT indexes
 
 ### A few notes
 
-1. Although specifing `K` requested results, the default `LIMIT` in RediSearch is 10, so for getting all the returned results, make sure to specify `LIMIT 0 {K}` in your command.
+1. Although specifying `K` requested results, the default `LIMIT` in RediSearch is 10, so for getting all the returned results, make sure to specify `LIMIT 0 {K}` in your command.
 
-2. By default, the resluts are sorted by their documents default RediSearch score. for getting the results sorted by similarity score, use `SORTBY {score field name}` as explained earlier.
+2. By default, the results are sorted by their documents default RediSearch score. for getting the results sorted by similarity score, use `SORTBY {score field name}` as explained earlier.
```

Comment:
By default, the results are sorted by their documents RediSearch score. If you would like to sort by vector similarity score, use `SORTBY {score field name}`. See examples below